src/main/java/com/adobe/jenkins/github_pr_comment_build/GithubHelper.java

```diff
@@ -0,0 +1,102 @@
+package com.adobe.jenkins.github_pr_comment_build;
+
+import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import hudson.model.Job;
+import java.io.IOException;
+import java.net.URI;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import jenkins.scm.api.SCMSource;
+import org.eclipse.egit.github.core.RepositoryId;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.service.CollaboratorService;
+import org.jenkinsci.plugins.github_branch_source.Connector;
+import org.jenkinsci.plugins.github_branch_source.GitHubSCMSource;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+/**
+ * client utility methods
+ */
+public class GithubHelper {
+
+    private final static Logger LOG = LoggerFactory.getLogger(GithubHelper.class);
+
+    private GithubHelper() {
+        // private
+    }
+
+    public static boolean isAuthorized(final Job<?, ?> job, final String author) {
+        GitHubClient client = getGitHubClient(job);
+        RepositoryId repository = getRepositoryId(job);
+        CollaboratorService collaboratorService = new CollaboratorService(client);
+
+        try {
+            boolean authorized = collaboratorService.isCollaborator(repository, author);
+            LOG.debug("User {} autorized: {}", author, authorized);
+            return authorized;
+
+        } catch (final IOException e) {
+            LOG.debug("Received an exception while trying to check if user {} is a collaborator of repository: {}",
```

Comment:
The args don't match up with the with formatting options.

Reply:
ok